config/member.yaml

```diff
@@ -24,6 +24,12 @@ orgs:
     members:
     - Mel0day
     - pkumza
+    - baiyutang
```

Comment:
how did you get the list?

Reply:
I check get the list at : https://github.com/orgs/kitex-contrib/people

Reply:
em. That means the correctness heavily relies on your line by line checking. Is there a way to fetch from API and do a comparison?

Reply:
Emm... I don't fully understand what you mean... You mean writing a tool to fetch the member list from GitHub API and update the this config file? From my understanding of GitOps, People from organization should update [this config file](https://github.com/kitex-contrib/test-infra/blob/main/config/member.yaml) manually and let prow run the peribolos job to update the org information periodically when config change.

As a result, the owners of org are supposed to update [this config file](https://github.com/kitex-contrib/test-infra/blob/main/config/member.yaml) and merge to upstream when they want to add a new member from outside.

Reply:
Yes, I agree. Every sub project need to update its own member config file manually, not exactly all the people from the  org member. @Summer-Field @Jeffwan